owslib/feature/schema.py

```diff
@@ -124,13 +124,17 @@ def _construct_schema(elements, nsmap):
     for element in elements:
         data_type = element.attrib["type"].replace(gml_key + ":", "")
         name = element.attrib["name"]
+        non_nillable = element.attrib["nillable"] == "false"
```

Comment:
I would replace this with something safer in the lines of
```python
non_nillable = element.attrib.get("nillable", "false") == "false"
```

On the basis that the `nillable` attribute on an xsd:element is optional, with a default of `false` as described in the [XSD schema](https://www.w3.org/2001/XMLSchema.xsd):
```xml
<xs:complexType name="element" abstract="true">
    <xs:annotation>
        <xs:documentation>
The element element can be used either at the top level to define an element-type binding globally, or within a content model to either reference a globally-defined element or type or declare an element-type binding locally. The ref form is not allowed at the top level.
        </xs:documentation>
    </xs:annotation>
    <xs:complexContent>
        <xs:extension base="xs:annotated">
            <xs:sequence>
                <xs:choice minOccurs="0">
                    <xs:element name="simpleType" type="xs:localSimpleType"/>
                    <xs:element name="complexType" type="xs:localComplexType"/>
                </xs:choice>
                <xs:group ref="xs:identityConstraint" minOccurs="0" maxOccurs="unbounded"/>
            </xs:sequence>
            <xs:attributeGroup ref="xs:defRef"/>
            <xs:attribute name="type" type="xs:QName"/>
            <xs:attribute name="substitutionGroup" type="xs:QName"/>
            <xs:attributeGroup ref="xs:occurs"/>
            <xs:attribute name="default" type="xs:string"/>
            <xs:attribute name="fixed" type="xs:string"/>
            <xs:attribute name="nillable" type="xs:boolean" use="optional" default="false"/>
            <xs:attribute name="abstract" type="xs:boolean" use="optional" default="false"/>
            <xs:attribute name="final" type="xs:derivationSet"/>
            <xs:attribute name="block" type="xs:blockSet"/>
            <xs:attribute name="form" type="xs:formChoice"/>
        </xs:extension>
    </xs:complexContent>
</xs:complexType>
```